examples/domainmodel/src/language-server/domain-model-index.ts

```diff
@@ -0,0 +1,36 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import { AstNodeDescription, DefaultAstNodeDescriptionProvider, LangiumDocument, LangiumServices, streamAllContents } from 'langium';
+import { DomainModelNameProvider } from './domain-model-naming';
+import { isPackageDeclaration, isType, PackageDeclaration } from './generated/ast';
+
+export class DomainModelDescriptionProvider extends DefaultAstNodeDescriptionProvider {
+
+    constructor(services: LangiumServices) {
+        super(services);
```

Comment:
This constructor can be omitted (supertype constructor is implicitly available).

Reply:
I added an explicit super constructor call after I noticed that `services` field was `undefined`